packages/terra-tag/tests/jest/Tag.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Tag from '../../src/Tag';
+
+describe('Tag', () => {
+  const defaultRender = <Tag text="default" />;
+
+  // Snapshot Tests
+  it('should render a tag', () => {
+    const tag = shallow(<Tag text="Normal" />);
+    expect(tag).toMatchSnapshot();
+  });
+
+  it('should render as an anchor when provided an href', () => {
+    const tag = shallow(<Tag text="href" href="MockHref" />);
+    expect(tag).toMatchSnapshot();
+  });
+
+  it('should render an icon and a text', () => {
```

Comment:
nit: rephrase to 'should render an icon and text'.

Reply:
updated it [here](https://github.com/cerner/terra-core/pull/1306/commits/6506e16da9f3b3d557d3c71bb7fa03a4874a99a9).